src/components/appRouter.js

```diff
@@ -508,42 +500,52 @@ class AppRouter {
 
     authenticate(ctx, route, callback) {
         const firstResult = this.firstConnectionResult;
-        if (firstResult) {
-            this.firstConnectionResult = null;
 
-            if (firstResult.State !== 'SignedIn' && !route.anonymous) {
-                this.handleConnectionResult(firstResult);
-                return;
-            }
+        this.firstConnectionResult = null;
+        if (firstResult && firstResult.State === 'ServerSignIn') {
```

Comment:
If user has a valid access token, we have `SignedIn` here and no wizard.
_I got this using "hackery" method to run wizard: change `system.xml/IsStartupWizardCompleted`. So probably safe under normal use._